metakb/database.py

```diff
@@ -340,12 +382,26 @@ def _add_document(tx, document: Dict):
                               f"{document}")
                 raise exception
 
+    def _get_ids_from_statement(self, statement, added_ids):
```

Comment:
if we start seeing slowdown in the upload process, we may want to change `added_ids` to a set once it's been built (or just make it a set from the start) for quicker 'is x in added_ids' operations